src/content/en/fundamentals/performance/resource-prioritization.md

```diff
@@ -0,0 +1,351 @@
+
+So what can you do if you find any resources that are marked with a different priority than the one you’d want?
+
+This article dives into three different declarative solutions, which are all
+relatively new `<link>` types. If your resources are crucial to the user
+experience but are being loaded at too low a priority, you can try fixing that
+in one of two ways: preload or preconnect. And if on the other hand you’d like
+the browser to fetch some resources only when it’s done dealing with everything
+else, try prefetch.
+
+Let’s look at all three!
+
+## Preload
+
+`<link rel="preload">` informs the browser that a resource will be needed as
+part of the current navigation, and that it should start getting fetched as soon
+as possible. Here’s how you use it:
+
+    <link rel="preload" as="script" href="super-important.js">
+    <link rel="preload" as="style" href="critical.css">
+
+Most of this is probably what you’d expect, except perhaps for the “as”
+attribute. This allows you to tell the browser the type of the resource you’re
+loading, so that it can be handled correctly; in fact, the browser will not use
+the preloaded resource unless the correct type is set. The resource will be
+loaded with the same priority as it would otherwise, but now the browser knows
+about it ahead of time, allowing for the download to start earlier.
+
+Note that `<link rel="preload">` is a compulsory instruction to the browser;
+unlike the other resource hints we’ll be talking about, it’s something the
+browser must do, rather than merely an optional hint. This makes it particularly
+important to test carefully, to insure that you’re not accidentally causing
+anything to fetch twice by using it, or fetching something that’s not needed.
+
+Resources that are fetched using `<link rel="preload">`, but not used by the
+current page within 3 seconds will trigger a warning in the Console in Chrome
+Developer Tools, so be sure to keep an eye out for these!
+
+<figure>
+  <div class="aspect-ratio"
+       style="width: 1050px; --aspect-ratio-w: 1050; --aspect-ratio-h: 244">
+    <img src="images/res-prio-timeout.png"
+    alt="An example of a preload timeout error in Chrome Developer Tools">
+  </div>
+</figure>
+
+### Use-case: Fonts
+
+Fonts are a great example of late-discovered resources that must be fetched,
+often sitting at the bottom of one of several CSS files loaded by a page.
+
+In order to reduce the amount of time the user has to wait for the text content
+of your site, as well as avoid jarring flashes between system fonts and your
+preferred ones, you can use `<link rel="preload">` in your HTML to let the
+browser know immediately that a font will be needed.
+
+    <link rel="preload" as="font" crossorigin="crossorigin" type="font/woff2" href="myfont.woff2">
+
+Note that the use of `crossorigin` here is important; without this attribute,
+the preloaded font will be ignored by the browser, and a new fetch will take
```

Comment:
Remove future tense: 'font is ignored by the browser, and a new fetch takes place'.

Reply:
Done.